README.md

```diff
@@ -998,3 +999,126 @@ go_proto_library(name, srcs, deps, has_services)
     </tr>
   </tbody>
 </table>
+
+### `go_embed_data`
+
+```bzl
+go_embed_data(name, src, srcs, out, package, var, flatten, string)
+```
+<table class="table table-condensed table-bordered table-params">
+  <colgroup>
+    <col class="col-param" />
+    <col class="param-description" />
+  </colgroup>
+  <thead>
+    <tr>
+      <th colspan="2">Attributes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>name</code></td>
+      <td>
+        <code>Name, required</code>
+        <p>A unique name for the go_embed_data rule.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>src</code></td>
+      <td>
+        <code>Label, optional</code>
+        <p>A single file to embed. This cannot be used at the same time as
+        <code>srcs</code>. The generated file will have a variable of type
+        <code>[]byte</code> or <code>string</code> with the contents of
+        this file.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>srcs</code></td>
+      <td>
+        <code>List of labels, optional</code>
+        <p>A list of files to embed. This cannot be used at the same time as
+        <code>src</code>. The generated file will have a variable of type
+        <code>map[string][]byte</code> or <code>map[string]string</code> with
+        the contents of each file. The map keys are relative paths the files
+        from the repository root. Keys for files in external repositories will
+        be prefixed with "external/repo/" where "repo" is the name of the
+        external repository.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>out</code></td>
+      <td>
+        <code>String, required</code>
+        <p>Name of the .go file to generated. This may be referenced by
+        other rules, such as <code>go_library</code>.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>package</code></td>
+      <td>
+        <code>String, required</code>
+        <p>Go package name for the generated .go file.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>var</code></td>
+      <td>
+        <code>String, required</code>
+        <p>Name of the variable that will contain the embedded data.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>flatten</code></td>
+      <td>
+        <code>Boolean, optional, defaults to false</code>
+        <p>If true and <code>srcs</code> is used, map keys are file base names
+        instead of relative paths.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>string</code></td>
```

Comment:
I wanted to make sure this is compatible with the internal version. Ideally we can unify them at some point.